<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>my_network</class>
 <widget class="QWidget" name="my_network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#00007f;&quot;&gt;Exit your account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Quit Account </string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="me_button">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>me</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notif_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>notiif</string>
    </property>
   </widget>
   <widget class="QPushButton" name="job_button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>job</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mynetwork_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>my network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="message_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>messeage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serach_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>search ID</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="request_button">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Requests</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="suggest_groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>1091</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Accounts you might be like it</string>
   </property>
   <widget class="QGroupBox" name="account_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>215</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_button_1">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="account_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>215</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_button_2">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="account_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>215</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_button_3">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="account_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>215</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_button_4">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="account_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>211</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="profile_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="follow_button_5">
     <property name="geometry">
      <rect>
       <x>69</x>
       <y>150</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Follow</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastname_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lastname</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="request_groupbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="profile_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_button_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastname_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>last name :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decline_button_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decline</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>email :</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_r">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="last_name_r">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="email_r">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="previous_pushbutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_pushbutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
